### PR TITLE
Stop emulation when render window is closed

### LINKED ISSUE #1110: Citra continues emulation even if the render window is closed.
#### Reproduction
- Open a game with citra (Qt)
- Make sure you are rendering to a separate window
- Close the render window

#### What I expected
In this situation, Citra should either halt emulation, or disable render-to-main-window.

#### What happens
Citra continues emulation, which can either be halted with the menu stop button or returned to rendering to main window.

### PR DESCRIPTION
**Pull request by LittleWhite-tb**
Fixes #1110

### CODE REVIEW

File: src/citra_qt/main.cpp [hunk 1]
```diff
@@ -268,6 +270,9 @@ void GMainWindow::BootGame(const std::string& filename) {
 }
 
 void GMainWindow::ShutdownGame() {
+    if (!emu_thread)
```
Comment (neobrain) [inline src/citra_qt/main.cpp]: Needs a comment explaining when this wouldn't be the case. If this is just a safety check to workaround broken design, please drop it; instead fix the true issue, or add a TODO about it.

File: src/citra_qt/bootmanager.h [hunk 2]
```diff
@@ -129,6 +131,10 @@ public slots:
     void OnEmulationStarting(EmuThread* emu_thread);
     void OnEmulationStopping();
 
+signals:
+    /// Emitted when the Windows is closed
+    void WindowClosed();
```
Comment (neobrain) [inline src/citra_qt/bootmanager.h]: Why not simply "Closed()"?

**Comment by LittleWhite-tb** [comment]
@neobrain I have rethink my code to remove the bad code. Now, I monitor the render window closing only when the emulation is started.

**Comment by bunnei** [comment]
LGTM

**Pull request merged**